imports/client/components/AccountForm.tsx

```diff
@@ -1,20 +1,50 @@
 import { Meteor } from 'meteor/meteor';
 import { Accounts } from 'meteor/accounts-base';
-import PropTypes from 'prop-types';
-import React from 'react';
-import Alert from 'react-bootstrap/lib/Alert';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
Is this going to muck with our ability to do lazy-loading (which I previously recall you mentioning required using just default imports?)

Reply:
Hmm. I don't think so. This mostly comes into play because TypeScript draws a [distinction](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) between a "default export" and an "exports object" that I don't entirely understand. The docs assert that they're incompatible, which I don't totally see, but I also don't have an especially deep understanding of how the different modules systems work.

In any case, I think we have several options here:
- It's not clear that deferring imports of React itself is especially valuable, since we won't be able to _do_ anything until it loads anyway. (And I think we can avoid needing this `*` import for our own modules)
- I think it's possible we can tune this behavior with the `allowSyntheticDefaultImports` setting in `tsconfig.json`